cmd/dcos/main.go

```diff
@@ -26,13 +29,45 @@ func main() {
 		Fs:         afero.NewOsFs(),
 	})
 
-	if len(os.Args) == 2 && os.Args[1] == "--version" {
+	if err := run(ctx, os.Args); err != nil {
```

Comment:
I don't know enough about how you propagate errors in the code base for logging purposes. Is it OK to throw away the error returned here and not log it? I assume you log it further down the call stack otherwise?

Reply:
Yeah this is handled by cobra, it will print errors by default when using the `RunE` method and `SilenceErrors` is not set to true.